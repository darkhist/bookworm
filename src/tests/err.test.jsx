import React from 'react';
import { shallow } from 'enzyme';

import Err from '../components/err/err';

describe('Err', () => {
  it('should render without crashing', () => {
    const err = shallow(<Err msg="" />);
    expect(err).toMatchSnapshot();
  });

  it('should render with "no results"', () => {
    const err = shallow(<Err msg="no results" />);
    expect(err).toMatchSnapshot();
  });

  it('should render with "oops. something went wrong"', () => {
    const err = shallow(<Err msg="oops. something went wrong" />);
    expect(err).toMatchSnapshot();
  });
});
