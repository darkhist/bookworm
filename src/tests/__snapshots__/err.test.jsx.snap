// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Err should render with "no results" 1`] = `
<styled.div>
  no results
  <br />
  <span
    aria-label="sad face emoji"
    role="img"
  >
    ðŸ˜¢
  </span>
</styled.div>
`;

exports[`Err should render with "oops. something went wrong" 1`] = `
<styled.div>
  oops. something went wrong
  <br />
  <span
    aria-label="sad face emoji"
    role="img"
  >
    ðŸ˜¢
  </span>
</styled.div>
`;

exports[`Err should render without crashing 1`] = `
<styled.div>
  <br />
  <span
    aria-label="sad face emoji"
    role="img"
  >
    ðŸ˜¢
  </span>
</styled.div>
`;
